patches/build-config-locales.gni.patch

```diff
@@ -1,12 +1,21 @@
 diff --git a/build/config/locales.gni b/build/config/locales.gni
-index 428136f8fcbda7f538a018932ecc2a04ff06132a..762d5538854c258735e8f5fe427ee105fa434b9e 100644
+index 37632eb876b207e6aa303796e9c4ffe788a612ae..7f3455d6a66308a7784306c282191ed613762aaa 100644
 --- a/build/config/locales.gni
 +++ b/build/config/locales.gni
+@@ -16,7 +16,7 @@ declare_args() {
+   # pass the "--translate-genders" flag into GRIT to generate 4 translation
+   # files per locale ("", "_FEMININE", "_MASCULINE", "_NEUTER") instead of
+   # one.
+-  translate_genders = true
++  translate_genders = is_android
```

Comment:
This change https://github.com/brave/brave-core/pull/29767/commits/8b40636df0a995972b6f8b35b077d573375817c5 disabled it for Android in our config. Can we just do the same for all platforms instead of patching?

Reply:
cc: @cdesouza-chromium

Reply:
Fixed in ea98322a827.